### PR TITLE
remove plot_url from statistical posts

### PR DESCRIPTION
**Pull request by jdamiba**

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
linspace still showing up in a couple of places

also the final example in http://localhost:4000/javascript/2d-density-plots/ should still be embedded

File: _posts/plotly_js/statistical/density-plots/2015-08-12-2D-density-histogram-slider-control.html [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
 name: 2D Histogram Contour Plot with Slider Control
-plot_url: https://jsfiddle.net/plotlygraphs/y9sdy76h/4/embedded/result,js,html/
```
Comment (nicolaskruchten) [inline _posts/plotly_js/statistical/density-plots/2015-08-12-2D-density-histogram-slider-control.html]: this one should stay

Comment (jdamiba) [reply _posts/plotly_js/statistical/density-plots/2015-08-12-2D-density-histogram-slider-control.html]: Re-added.

**Comment by nicolaskruchten** [comment]
💃

**Pull request merged**